lib/node_modules/@stdlib/array/complex64/README.md

```diff
@@ -2098,6 +2098,68 @@ var count = context.count;
 // returns 2
 ```
 
+<a name="method-sort"></a>
+
+#### Complex64Array.prototype.sort( compareFcn )
+
+Sorts an array in-place.
+
+```javascript
+var realf = require( '@stdlib/complex/realf' );
+var imagf = require( '@stdlib/complex/imagf' );
+
+function compare( a, b ) {
+    return realf( a ) - realf( b );
```

Comment:
For enforcing the sort order, let's follow NumPy (see https://numpy.org/devdocs/reference/generated/numpy.sort.html) and it's sort order.

E.g.,

```javascript
var re1;
var re2;
var im1;
var im2;

re1 = realf( a );
re2 = realf( b );
if ( re1 < re2 ) {
    return -1;
}
if ( re1 > re2 ) {
    return 1;
}
// re1 === re2
im1 = imagf( a );
im2 = imagf( b );
if ( im1 < im2 ) {
    return -1;
}
if ( im1 > im2 ) {
    return 1;
}
return 0;
```

This applies here and elsewhere.